### PR TITLE
Feature: Added option to pin recycle bin to the start menu

### PR DESCRIPTION
**Pull request by Hark64**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue #6230

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.
image
image

### CODE REVIEW

**Comment by yaira2** [comment]
Can you verify the functionality works?

**Comment by yaira2** [comment]
Thank you for your contribution, we still haven't decided if we want to include this functionality so we're putting this on hold.

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Helpers/ContextFlyoutItemHelper.cs [hunk 1]
```diff
@@ -992,6 +995,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
 					Command = commandsViewModel.UnpinItemFromStartCommand,
 					ShowOnShift = true,
 					ShowItem = selectedItems.All(x => !x.IsShortcut && (x.PrimaryItemAttribute == StorageItemTypes.Folder || x.IsExecutable) && !x.IsArchive && x.IsItemPinnedToStart),
+					ShowInRecycleBin = true,
```
Comment (gave92) [inline src/Files.App/Helpers/ContextFlyoutItemHelper.cs:998]: I don't think we want this one, this is about items contained in the recycle bin not the recycle bin folder.

Comment (yaira2) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:998]: @Hark64 can you open a PR to resolve?

Comment (Hark64) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:998]: Sorry about that. I opened PR #10945 to address this.

**Pull request merged**